project/src/main/webapp/create-event.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js"></script>
+    <!-- Bootstrap CDN links -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+      />
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+      ></script>
+  </head>
+  <body>
+    <br><br>
+    <h3 style="text-align:center" > Create an Event </h3>
```

Comment:
I think these styling statements should go in the css so it's easier to search for and edit later

Reply:
i wanted to stylize these inline because I dont want to override styles like <h3> if someone else uses them and wants them default / to look like something else -- let me know if that makes sense 

same with like bootstrap buttons etc

Reply:
We can totally discuss this in chat - I personally think that we should keep styles in a CSS file with IDs/classes as needed. As for whether we want to do a 1:1 with CSS/HTML files, that might be a best practices question?

Reply:
resolved. done